scrapers/AdultPrime.yml

```diff
@@ -0,0 +1,30 @@
+name: "AdultPrime"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - adultprime.com/studios/video/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h2[contains(@class,"update-info-title")]/text()
+      Details: //p[contains(@class,"ap-limited-description-text")]/text()

```

Comment:
This gets only a part of the description, a post process and concat is needed.
Is the description good/useful to scrape though? Unless i am doing something wrong it seems that the description is meant for the studio and not the scenes, so for every scene of the same studio we get the same description?
eg `https://adultprime.com/studios/video/122995` `https://adultprime.com/studios/video/123113`

Reply:
removed as requested; the site might have changed since I threw this together, noticed it wasn't in the repo so I added it.

however it seems that scenes either have a custom description or are given a generic one there are some that are unique
e.x. `https://adultprime.com/studios/video/27075`